packages/node/src/index.ts

```diff
@@ -175,7 +175,9 @@ async function compile(
     return source;
   }
 
-  const conditions = isEdgeFunction ? ['worker', 'browser'] : undefined;
+  const conditions = isEdgeFunction
+    ? ['edge-light', 'browser', 'module', 'import', 'require']
```

Comment:
Why `require()`? Isn't that going to include files that don't work on the Edge Runtime?

Reply:
CJS does still work in Edge runtime.